docs/Python-API.md

```diff
@@ -111,8 +111,8 @@ A `BaseEnv` has the following methods:
   terminates the communication.
 - **Behavior Specs : `env.behavior_specs`** Returns a Mapping of
   `BehaviorName` to `BehaviorSpec` objects (read only).
-  A `BehaviorSpec` contains information such as the observation shapes, the
-  action type (multi-discrete or continuous) and the action shape. Note that
+  A `BehaviorSpec` contains information such as the observation shapes and the
```

Comment:
@vincentpierre @andrewcoh Can you review this part and make sure my changes are accurate?

Reply:
Currently the behavior spec just contains those two things, left a suggestion to just explicitly say that.